app/vmalert/datasource/init.go

```diff
@@ -74,6 +75,9 @@ func Init(extraParams url.Values) (QuerierBuilder, error) {
 	if *addr == "" {
 		return nil, fmt.Errorf("datasource.url is empty")
 	}
+	if queryTimeAlignment != nil {
```

Comment:
i think we should check for `false` (non-default value) here. Otherwise, it will be printed for all cases.
We also should add deprecation notice to the flag description. And make sure it was updated in readme (mentions and usage)

Reply:
Right, fixed and added.